File: diversos/teste-velocidade-arquivos/teste.py
def search_file(file_path, target_string):
    with open(file_path, 'r') as file:
        for line in file:
            if target_string in line:
                print('String found!')
                return True
        print('String not found.')
        return False

result = search_file('./teste.txt', '|9999|118247|')

File: diversos/teste-velocidade-arquivos/teste.js
const fs = require('fs');

fs.readFile('./teste.txt', 'utf8', (err, data) => {
  if (err) {
    console.error(err);
    return;
  }
  
  if (data.includes('|9999|118247|')) {
    console.log('String found!');
  } else {
    console.log('String not found.');
  }
});
